config/metadata/core_metadata.yaml

```diff
@@ -1,4 +1,8 @@
 attributes:
+  human_readable_type:
```

Comment:
doesn't this add `human_readable_type` to `Hyrax::Schema(:core_metadata)`??

have you explored the impact of this? would it be better to put this in `ResourceIndexer` alongside `alternate_ids`?

Reply:
Yes, I was just looking at this again as I was adding has_model and agree it is better in ResourceIndexer.